brave/src/test/java/brave/test/TestSpanHandler.java

```diff
@@ -0,0 +1,44 @@
+package brave.test;
+
+import brave.handler.MutableSpan;
+import brave.handler.SpanHandler;
+import brave.propagation.TraceContext;
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+import java.util.stream.Stream;
+
+// copy of what's in brave-tests
+public final class TestSpanHandler extends SpanHandler implements Iterable<MutableSpan> {
```

Comment:
this is a preview of what will be in brave-tests. The "reporter" which was there will be renamed to IntegrationTestSpanHandler